RELEASE/scripts/autoscend/auto_edTheUndying.ash

```diff
@@ -1434,7 +1434,12 @@ boolean LM_edTheUndying()
 	{
 		return true;
 	}
-	// Airship is 1.5 Ka or 1.8 Ka with the construct banished so third highest priorty after the war
+	// Mob of Zeppelin Protestors is a 1.8 Ka zone. Now that pirate path is gone we actually want to do it.
+	if(L11_redZeppelin())
+	{
+		return true;
+	}
+	// Airship is 1.5 Ka or 1.8 Ka with the construct banished so it is the next priority zone
```

Comment:
No. Remove this (and try scrolling down the page and reading the comments I leave for a change).

Reply:
I do read. I managed to miss that one somehow. Sorry. will revert.

Reply:
reverted on ac52148